inspec.yml

```diff
@@ -0,0 +1,611 @@
+    type: Boolean
+    value: true
+    profile: redhat-enterprise-linux-8-stig-baseline
+
+  # SV-230537
+  - name: ipv6_enabled
+    desc: Set to 'true' if IPv6 is enabled on the system.
+    type: Boolean
+    value: true
+    profile: redhat-enterprise-linux-8-stig-baseline
+
+  # SV-230493
+  - name: camera_installed
+    desc: Device or system does not have a camera installed.
+    type: Boolean
+    value: true
+    profile: redhat-enterprise-linux-8-stig-baseline
+
+  # SV-230503
+  - name: bluetooth_installed
+    desc: 'Device or operating system has a Bluetooth adapter installed'
+    type: Boolean
+    value: true
+    profile: redhat-enterprise-linux-8-stig-baseline
+
+  # SV-230242
+  - name: known_system_accounts
+    desc: System accounts that support approved system activities.
+    type: Array
+    value: 
+      - 'root'
+      - 'bin'
+      - 'daemon'
+      - 'adm'
+      - 'lp'
+      - 'sync'
+      - 'shutdown'
+      - 'halt'
+      - 'mail'
+      - 'operator'
+      - 'nobody'
+      - 'systemd-bus-proxy'
+      - 'dbus'
+      - 'polkitd'
+      - 'postfix'
+      - 'sssd'
+      - 'chrony'
+      - 'systemd-network'
+      - 'sshd'
+      - 'ntp'
+    profile: redhat-enterprise-linux-8-stig-baseline
+
+  - name: smart_card_status
+    desc: Smart card status (enabled or disabled)
+    type: String
+    value: 'enabled'
+    profile: redhat-enterprise-linux-8-stig-baseline
+
+  # SV-230263
+  - name: file_integrity_tool
+    desc: Name of tool
+    type: String
+    value: 'aide'
+    profile: redhat-enterprise-linux-8-stig-baseline
+
+  # SV-230484
+  - name: authoritative_timeserver
+    desc: Timeserver used in /etc/chrony.conf
+    type: String
+    value: 0.us.pool.ntp.mil
+    profile: redhat-enterprise-linux-8-stig-baseline
+
+  # SV-230537
+  - name: non_removable_media_fs
+    desc: File systems listed in /etc/fstab which are not removable media devices
+    type: Array
+    value: ["/", "/tmp", "none", "/home"]
+    profile: redhat-enterprise-linux-8-stig-baseline
+
+  # SV-230230
+  - name: private_key_files
+    desc: List of full paths to private key files on the system
+    type: Array
+    value: []
+    profile: redhat-enterprise-linux-8-stig-baseline
+
+  #SV-230229
+  - name: root_ca_file
+    desc: Path to an accepted trust anchor certificate file (DoD)
+    type: String
+    value: "/etc/sssd/pki/sssd_auth_ca_db.pem"
+    profile: redhat-enterprise-linux-8-stig-baseline
+
+  #SV-230373
+  - name: days_of_inactivity
+    desc: Maximum number of days if account inactivity before account lockout
+    type: Numeric
+    value: 35
+    profile: redhat-enterprise-linux-8-stig-baseline
+
+  - name: temporary_accounts
+    desc: Temporary user accounts
+    type: Array
+    value: []
+    profile: redhat-enterprise-linux-8-stig-baseline
+
+  - name: maxlogins_limit
+    desc: Amount of max logins allowed
+    type: String
+    value: '10'
+    profile: redhat-enterprise-linux-8-stig-baseline
```

Comment:
For `High` the value needs to be overridden to `1`
Consider making the data type integer in the baseline (log an issue if short on time)

Reply:
Both High and HIgh-HVA have a value of 1 in ARS 5

Reply:
Done https://github.com/CMSgov/cms-ars-5.0-red-hat-enterprise-linux-8-stig-overlay/pull/1/commits/f70f5924ecbfbdbdcf6b52e92f6ede9e9f750356